splice_machine/src/test/java/com/splicemachine/derby/impl/load/HdfsImportIT.java

```diff
@@ -905,6 +905,44 @@ public void testNoSeparatorDateTime() throws Exception {
         Assert.assertTrue("import failed!", results.size() == 1);
     }
 
+    @Test
+    public void testDaylightSavingsTimeProblematicDateTime() throws Exception {
+        methodWatcher.executeUpdate("delete from " + spliceTableWatcher9);
+        PreparedStatement ps = methodWatcher.prepareStatement(format("call SYSCS_UTIL.IMPORT_DATA(" +
+                        "'%s'," +  // schema name
+                        "'%s'," +  // table name
+                        "null," +  // insert column list
+                        "'%s'," +  // file path
+                        "','," +   // column delimiter
+                        "'%s'," +  // character delimiter
+                        "'yyyyMMdd''T''HH:mm:ss.SSSSSS'," +  // timestamp format
+                        "null," +  // date format
+                        "null," +  // time format
+                        "%d," +    // max bad records
+                        "'%s'," +  // bad record dir
+                        "null," +  // has one line records
+                        "null)",                    // char set
+                spliceSchemaWatcher.schemaName, TABLE_9,
+                getResourceDirectory() + "no_separator_datetimes_dst.csv",
+                "\"",                           0,
+                BADDIR.getCanonicalPath()));
+        ps.execute();
+
+        ResultSet rs = methodWatcher.executeQuery(format("select * from %s.%s", spliceSchemaWatcher.schemaName,
+                TABLE_9));
+        List<String> results = Lists.newArrayList();
+        while (rs.next()) {
+            Timestamp order_date = rs.getTimestamp(1);
+            assertNotNull("order_date incorrect", order_date);
+            // The system may or may not be in a time zone that observes daylight savings.
+            Assert.assertTrue(String.format("Unexpected timestamp value: %s", order_date.toString()),
+                                order_date.toString().equals("2012-03-11 03:05:12.123456") ||
```

Comment:
While we're here, should we also verify "fall back" dates?  Or are we doing that somewhere else?

Reply:
Those were handled by SPLICE-2264.  There's a test for it here:  https://github.com/splicemachine/spliceengine/pull/2404/files#diff-83845c0e11fdf371b3bcfd3456fa9d33